package.json

```diff
@@ -9,8 +9,11 @@
   "dependencies": {
     "api-spec-converter": "2.7.32"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "@stoplight/spectral": "^5.7.1"
+  },
   "scripts": {
-    "convert": "api-spec-converter --from=openapi_3 --to=swagger_2 --syntax=json docs/openapi.json"
+    "convert": "api-spec-converter --from=openapi_3 --to=swagger_2 --syntax=json docs/openapi.json",
```

Comment:
I think we can remove this right? not used.

Reply:
I think it's used when launching the gitpod.